pandas/tests/frame/methods/test_rename.py

```diff
@@ -53,10 +53,10 @@ def test_rename(self, float_frame):
         # gets sorted alphabetical
```

Comment:
Can you remove this comment?

Reply:
Just fixed both of these! Thanks for taking a look.